common/step_download.go

```diff
@@ -61,7 +67,7 @@ func (s *StepDownload) Run(_ context.Context, state multistep.StateBag) multiste
 		}
 	}
 
-	ui.Say(fmt.Sprintf("Downloading or copying %s", s.Description))
+	ui.Say(fmt.Sprintf("Downloading, copying or inplace referencing %s", s.Description))
```

Comment:
I agree with @arizvisa that this seems redundant now. I'd still like a logline here because it signals that we're entering the main downloader code, but what about just saying something generic like ("Retrieving %s", s.Description) instead?